web3_proxy/src/frontend/mod.rs

```diff
@@ -160,6 +161,14 @@ pub async fn serve(port: u16, proxy_app: Arc<Web3ProxyApp>) -> anyhow::Result<()
         )
         .route("/user/stats/detailed", get(users::user_stats_detailed_get))
         .route("/user/logout", post(users::user_logout_post))
+        .route("/admin/modify_role", get(admin::admin_change_user_roles))
+        .route("/admin/imitate-login/:user_address", get(admin::admin_login_get))
+        .route(
+            "/user/imitate-login/:user_address/:message_eip",
```

Comment:
i think this route should be `/admin/imitate-login/:user_address/:message_eip` to match the one before it